### ISSUE TITLE
Latest recommended Windows 10 version?

### ISSUE THREAD

**Issue by lkuhn65** [MicrosoftDocs/memdocs#497]
[Enter feedback here]

In the "Create a Desktop Analytics deployment plan" section it says "Products and versions: Select the Windows product, and the latest available recommended version. For example, Windows 10, version 1809 (recommended)" - shouldn't 1809 be something newer by now, 1909 or 2004? - or maybe just reword this to say "See Windows 10 documentation to see what the current recommended version for broad enterprise deployment is as of the time you are following this tutorial. https://docs.microsoft.com/en-us/windows/release-information/"
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a6c3e494-65a1-6c1b-118d-963e1ac8ca06
* Version Independent ID: 776ca6de-38a8-3c95-4910-9e7cffec0661
* Content: [Tutorial - Deploy Windows 10 - Configuration Manager](https://docs.microsoft.com/en-us/mem/configmgr/desktop-analytics/tutorial-windows10)
* Content Source: [memdocs/configmgr/desktop-analytics/tutorial-windows10.md](https://github.com/MicrosoftDocs/memdocs/blob/master/memdocs/configmgr/desktop-analytics/tutorial-windows10.md)
* Product: **configuration-manager**
* Technology: **configmgr-analytics**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

**Comment by aczechowski** [comment]
Thanks @lkuhn65 for the feedback! Yes, that should definitely change, thanks for catching it. I'll use the same language as the article, [How to create deployment plans](https://docs.microsoft.com/mem/configmgr/desktop-analytics/create-deployment-plans):

> Choose which Windows 10 version to deploy. Microsoft recommends creating deployment plans that use the most recent version.

I didn't find any other instance of that older version in a similar context. I'll make the change now, it will publish live later this afternoon.

**Comment by lkuhn65** [comment]
Sounds great!

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by aczechowski** [comment]
(quick admin edit to remove your email address from the details of your last response, so it's not publicly available. that's the catch with replying via email. 😞 )

And FYI, we're having a backend publishing issue right now. Once that's fix and this change goes through, then this issue should auto close. The change will be live with the next publishing cycle, which is twice every business day.
